java-frontend/src/test/java/org/sonar/java/bytecode/cfg/testdata/MethodLookup.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * SonarQube Java
+ * Copyright (C) 2012-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.java.bytecode.cfg.testdata;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.NoSuchElementException;
+
+/**
+ * Test data for {@link org.sonar.java.bytecode.se.MethodLookup}
+ */
+public class MethodLookup extends SuperClass implements Iface {
```

Comment:
I think this class name is a bit confusing, it is a test indeed but same class name as production one distinguished only by package name is not ideal. Renaming it to LookupTestMethods or MethodLookupTestData should do the trick and avoid confusion ?